### PR TITLE
Allow for nested aggregated projects

### PR DESCRIPTION
**Pull request by chrisbenincasa**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    setBuildName(version)
+
+    stage('Build') {
+        sbt.execute("clean +compile")
+    }
+
+    stage('Test') {
+        try {
+          sbt.execute("scripted")
+        }
```
Comment (richid) [inline Jenkinsfile:20]: Is the `try` giving us anything here?

File: scripted.sbt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ScriptedPlugin.scriptedSettings
 
 scriptedLaunchOpts := { scriptedLaunchOpts.value ++
-  Seq("-Xmx1024M", "-XX:MaxPermSize=256M", "-Dplugin.version=" + version.value)
+  Seq("-Xmx2G", "-XX:MaxPermSize=512M", "-Dplugin.version=" + version.value)
```
Comment (richid) [inline scripted.sbt:4]: Do we need to set `MaxPermSize` at all? Assuming this requires J8...

**Review by richid — APPROVED** [review]
Just a couple nits
.
.
.
.
.
.
.
.
hell yeah

**Review by devshorts — APPROVED** [review]
Looks good to me, make sure you comment in the final commit message how you tested this so future debuggers know what to do

**Pull request closed without merging**